keyboards/ymd96/ymd96.h

```diff
@@ -77,4 +77,22 @@ K411, K410, K409, K408, K404, K405, K406, K407, K403, K402, K401, K400  \
 }
 
 
+#define KEYMAP_AnthonyWharton( \
```

Comment:
If you're intending to add this layout/matrix for only your own use, this whole #define should go in your keymap-level, `config.h`, IIRC. @drashna please advise.

If you're intending this to be available to anyone with a YMD96, your PR will need some further changes that we can walk you through. Alternatively, we could collectively decide to merge it as-is, but some of the changes you've made will need to be updated further, including changes to your personal keymap files. The YMD96's codebase right now is based on older standards and not currently compliant with QMK best practices, but hasn't been updated yet.

Reply:
Apologies, I'm happy to move that over to the keymap-level folder, expect a commit in just a moment. 

If you could advise on what I'd need to do to make this publicly accessible I'd be happy to do that in a future PR/in future here.